### ISSUE TITLE
Confuse with labeling

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework5G3#30]
@uarif Are you sure, is that the time for the x-axis label in the clean.py file

**Comment by poojashresthacode** [comment]
I think it should be frequency,.

**Comment by uarif** [comment]
You are right, it should be frequency, but I am thinking about an appropriate unit, and I will fix that later. Thank you nyway.
